### DISCUSSION TITLE
Rede Globo SP

### DISCUSSION THREAD

**Discussion by lgissoni** [iptv-org/community#708] [Q&A]
**Channel Name:** Rede Globo SP
**Channel ID:** TVGloboSaoPaulo.br
**Country:** Brazil
**Language:** Portuguese
**Website:** https://redeglobo.globo.com/sao-paulo/
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by AntiPontifex** [comment]
Defunct: please close.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19696
